vendor.conf

```diff
@@ -1,11 +1,11 @@
-k8s.io/kubernetes 7488d1c9210e60aef9ad49f07cb5d8a24152db88 https://github.com/kubernetes/kubernetes
+k8s.io/kubernetes fe0e80e8dafcdfde9778380f40346bfa75bad0f7 https://github.com/kubernetes/kubernetes
```

Comment:
Let us switch to the latest version in release-1.0 branch and also change to the same in the e2e tests https://github.com/kubernetes-incubator/cri-o/blob/master/contrib/test/integration/main.yml#L121

Reply:
https://github.com/kubernetes/kubernetes/commits/release-1.10

Reply:
Updated vendor and yaml to use the release 1.10 branch.
Added integration test.